### PR TITLE
Update Jenkins version to current LTS

### PR DESCRIPTION
**Pull request by lachie83**
Duplicates -- https://github.com/kubernetes/charts/pull/102

cc @nrshrivatsan

### CODE REVIEW

**Comment by lachie83** [comment]
@viglesiasce Can you confirm how the master-image gets pushed to the image repo? I'm currently assumed the tag

File: stable/jenkins/master-image/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.7.4
+FROM jenkins:2.19.1
```
Comment (naveensrinivasan) [inline stable/jenkins/master-image/Dockerfile]: There was another release  ,which  is `2.19.2` 
https://jenkins.io/changelog-stable/

**Comment by lachie83** [comment]
@viglesiasce Can you please confirm how the master images gets built and put in the repo. I need to confirm the tags match.

**Comment by viglesiasce** [comment]
@lachie83 currently it is built and pushed manually which is not ideal. I have some ideas on fixing this but not time atm to implement them. Will pull this down and push this image then merge the PR.

**Comment by viglesiasce** [comment]
Image pushed. Testing now.

**Comment by viglesiasce** [comment]
Working as expected. Thanks @lachie83!

**Comment by prydonius** [comment]
@k8s ok to test

**Comment by viglesiasce** [comment]
@k8s ok to test

**Pull request merged**